### ISSUE TITLE
Radio buttons  present under 'Settings' blade like ‘Custom and Unlimited’ along with its label ‘Page options’ are not enclosed in fieldset/legend tag.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1034]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1239420

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to 'Settings' menu item and select it.
Navigate through all the controls under Settings blade.
Press tab key to navigate over radio buttons  ‘Custom' and 'Unlimited’.
Press F12 key to open the Developers tool and inspect the code.
Verify the issue.

Actual Result:
Radio buttons  present under 'Settings' blade like ‘Custom and Unlimited’ along with its label ‘Page options’ are not enclosed in fieldset/legend tag and  it is not defined under label attribute which is incorrect.

Expected Result:
Radio buttons  present under 'Settings' blade like ‘Custom and Unlimited’ along with its label ‘Page options’ are not enclosed in fieldset/legend tag.
Screen reader should announce Page options, 'Custom' radio button checked one of two to the the end user.


Note:
Similar issue exits for 'Unlimited' radio controls under same section.

![image](https://user-images.githubusercontent.com/8519407/132719243-055b095b-ab02-4457-bd11-0395c92f639f.png)
